app/commands/content_module_cli.rb

```diff
@@ -55,20 +55,6 @@ def slides
     snapshotter.generate
   end
 
-
```

Comment:
I think this works?
I guess I had reviewed the module text/video content UI using it.

Reply:
I think it should work...